Makefile

```diff
@@ -33,7 +33,6 @@ APP = gatekeeper
 SRCS-y := main/main.c
 
 # Functional blocks.
-SRCS-y += catcher/main.c
```

Comment:
It seems the commit order here doesn't match the real order that I committed.

Here is the order that I committed:

1. lib: fixed typo in acl.c
2. bp: drop the old code of bp
3. rt: drop the old code of rt
4. catcher: drop the old code of catcher
5. gk: implement bp and rt in gk blocks